vllm/model_executor/layers/rotary_embedding.py

```diff
@@ -157,8 +157,6 @@ def forward_cuda(
     ) -> Tuple[torch.Tensor, torch.Tensor]:
         from vllm import _custom_ops as ops
 
-        self.cos_sin_cache = self.cos_sin_cache.to(query.device,
```

Comment:
to keep the equivalence, we should move this line into `__init__`

Reply:
@youkaichao Oh can you please explain more? IIUC, this is purely redundant as the cache is initialized in GPU from the first place.

Reply:
there's one more line below, `dtype=query.dtype`

Reply:
oh I didn't see that. thanks.